collective/easyform/profiles/default/controlpanel.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0"?>
 <object name="portal_controlpanel">
- <configlet title="EasyForm" action_id="EasyForm" appId="EasyForm"
+ <configlet title="easyform" action_id="easyform" appId="easyform"
```

Comment:
At least `title` should start with uppercase, because that's displayed in the UI.